cognite_toolkit/cdf_tk/bootstrap.py

```diff
@@ -32,34 +33,34 @@
 
 @dataclass
 class AuthVariables:
-    cluster: str
-    project: str
-    token: str
-    client_id: str
-    client_secret: str
-    cdf_url: str = None
-    token_url: str = None
-    tenant_id: str = None
-    audience: str = None
-    scopes: str = None
+    cluster: str | None
+    project: str | None
+    token: str | None
+    client_id: str | None
+    client_secret: str | None
+    cdf_url: str | None = None
+    token_url: str | None = None
+    tenant_id: str | None = None
+    audience: str | None = None
+    scopes: str | None = None
     ok: bool = False
     info: str = ""
+    error: bool = False
+    warning: bool = False
 
 
 def get_auth_variables(interactive: bool = False, verbose: bool = False) -> AuthVariables:
     vars = AuthVariables(
-        cluster=os.environ.get("CDF_CLUSTER", None),
-        project=os.environ.get("CDF_PROJECT", None),
-        token=os.environ.get("CDF_TOKEN", None),
-        client_id=os.environ.get("IDP_CLIENT_ID", None),
-        client_secret=os.environ.get("IDP_CLIENT_SECRET", None),
-        cdf_url=os.environ.get("CDF_URL", None),
-        tenant_id=os.environ.get("IDP_TENANT_ID", None),
-        audience=os.environ.get("IDP_AUDIENCE", None),
-        scopes=os.environ.get("IDP_SCOPES", None),
```

Comment:
Any particular reason for removing the explicitness of setting the var to None? It's default, but the less you need to remember about a specific function, the better IMO